config/params/mainnet_config.go

```diff
@@ -21,8 +21,8 @@ const (
 	genesisForkEpoch = 0
 	// Altair Fork Epoch for mainnet config.
 	mainnetAltairForkEpoch = 74240 // Oct 27, 2021, 10:56:23am UTC
-	// Placeholder for the merge epoch until it is decided
-	mainnetBellatrixForkEpoch = math.MaxUint64
+	// Bellatrix Fork Epoch for mainnet config.
+	mainnetBellatrixForkEpoch = 144896 // Sept 6, 2022, 11:34:47am UTC
```

Comment:
Checks out. Did 

genesis_time=1606824023
bellatrix_fork_epoch=144896
seconds_since_genesis=bellatrix_fork_epoch*32*12
genesis_time+seconds_since_genesis=1662464087

Put it into a unix converter and got
<img width="737" alt="Screen Shot 2022-08-17 at 11 15 15 AM" src="https://user-images.githubusercontent.com/5572669/185177328-098eba1a-023d-4969-b6f5-40c86319e63e.png">